### ISSUE TITLE
Syntax error?

### ISSUE THREAD

**Issue by j0hnsmith** [less/less.ruby#94]
This should generate a syntax error but doesn't (1.1.13)

```
#foo {
  font-size: 10px; 
  text-align: center: // s/be ; not :
}
#bar {
  height: 100px; 
}
```

Not really sure if it's a problem or not.

**Comment by j0hnsmith** [comment]
I guess this should also be a syntax error

```
p {
    margin-botton: 20px; // s/be margin-bottom
}
```

**Comment by cloudhead** [comment]
well `margin-botton` isn't a syntax error, it's just a property css doesn't support : ) — I think your browser should be able to tell you that.

**Comment by j0hnsmith** [comment]
Of course, it's not for less to decide what are valid properties and what aren't.

This is a syntax error though?
    text-align: center: // s/be ; not :

**Comment by cloudhead** [comment]
yea, that is, I'm gonna look into why it's passing through.

**Comment by vicalejuri** [comment]
What about this case?
filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(...);

I guess you should check for ; only on the EOL. Otherwise, you may block some
special uses of css (specially filters for IE).
